lib/server.js

```diff
@@ -111,7 +111,7 @@ function Server (torrent, opts) {
     function serveOptionsRequest () {
       res.statusCode = 204 // no content
       res.setHeader('Access-Control-Max-Age', '600')
-      res.setHeader('Access-Control-Allow-Methods', 'GET,HEAD,PUT,PATCH,POST,DELETE')
+      res.setHeader('Access-Control-Allow-Methods', 'GET,HEAD')
```

Comment:
lgtm - do you actually need HEAD though?

Reply:
We do currently support HEAD, so it needs to stay in this header if we want it to keep working. You can see how we handle it here: https://github.com/webtorrent/webtorrent/blob/8c8613d6bcb2a48a7aa996ffb08d379c3e20bea0/lib/server.js#L209

I don't think it hurts to keep it.